src/signalo/value_lists/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+
+class OfficialSignType(models.Model):
+    id = models.CharField(primary_key=True, max_length=10)
+    active = models.BooleanField(default=True)
+    value_de = models.CharField(max_length=255, null=True, blank=True)
+    value_fr = models.CharField(max_length=255, null=True, blank=True)
+    value_it = models.CharField(max_length=255, null=True, blank=True)
+    value_ro = models.CharField(max_length=255, null=True, blank=True)
+    description_de = models.TextField(null=True, blank=True)
+    description_fr = models.TextField(null=True, blank=True)
+    description_it = models.TextField(null=True, blank=True)
+    description_ro = models.TextField(null=True, blank=True)
+    img_de = models.FileField(upload_to="official_signs/de")
```

Comment:
in many cases, the images are the same from one language to another.
Can we easily avoid file duplication?
Otherwise, leave a comment as a fixme.

Reply:
Good catch? In many or in _all_ cases, though? Seeing from `official_signs.json` it looks like in fact all images are cross-referenced between distinct languages. So in fact we can just define: `img = models.FileField(upload_to="official_signs")` and leave it to the `description_<lang>` field to deal with translating things.